backend/utility/emailTemplates.js

```diff
@@ -0,0 +1,28 @@
+// This function will be used to generate a template which will be utilized in all email types
+// module.exports.generateHtml(data){};
+
+module.exports.getMailTemplate = (type) => {
+  const templates = {
+    'INVITE-ADMIN': {
+      // html:"",
+      subject: '[INVITATION] Invited to become an admin',
+      text:
+        'Hi $name$ \n We would love for you to join us at HITK Tech Community as an admin.\n\tHITK Tech Community is a platform built by the students and for the students with the main intent of increasing awareness towards plethora of opportunities and internships in tech all around/over the year. This will not only give practical work experience/exposure to students, but will also help everyone to know and grab their required opportunities in time!\nClick on $link$ for further action',
+    },
+  };
+  return templates[type];
+};
+
+module.exports.populateTemplate = (text, data) => {
```

Comment:
Good job on this! Kudos! but can you now find something better? Something like an industry-standard thing? The reason being if you use regex then if we cannot guarantee performance. There exist multiple solutions for such problems let me know if you want to give it a try. It will help you gain the skills to deep dive into multiple approaches. Take today's time to look for solutions, will complete this PR by EOD. Let me know if you need help

Reply:
Thanks,
I would be glad to improve the approach but cannot work on this for some time. I am an AKTU student and my semester exams are going on, so its not possible to finish the work by EOD.

Reply:
oh, can you share the timeline, till when you are blocked?

Reply:
I have exams on 17,19 and 22 march. I will resume the program on 19